uber/automated_emails.py

```diff
@@ -12,7 +16,7 @@ class AutomatedEmail:
 
     def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
         self.model, self.template, self.needs_approval = model, template, needs_approval
-        self.subject = subject.format(EVENT_NAME=c.EVENT_NAME)
+        self.subject = subject.format(EVENT_NAME=c.EVENT_NAME, EVENT_NAME_AND_DATE=c.EVENT_NAME_AND_DATE)
```

Comment:
Suggestion: replace ``{EVENT_NAME_AND_DATE}`` with ``{EVENT}`` since it's something that we're going to be repeating hundreds of times, so using the full form is kind of a lot.